sim/druid/maul.go

```diff
@@ -0,0 +1,106 @@
+package druid
+
+import (
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+func (druid *Druid) registerMaulSpell(realismICD *core.Cooldown) {
+	flatBaseDamage := 128.0
+	rageCost := 15 - float64(druid.Talents.Ferocity)
+
+	switch druid.Ranged().ID {
+	case IdolOfBrutality:
+		rageCost -= 3
+	}
+	rageMetrics := druid.NewRageMetrics(core.ActionID{SpellID: 431446})
+
+	druid.Maul = druid.RegisterSpell(Bear, core.SpellConfig{
+		ActionID:    core.ActionID{SpellID: 9881},
+		SpellSchool: core.SpellSchoolPhysical,
+		SpellCode:   SpellCode_DruidMaul,
+		DefenseType: core.DefenseTypeMelee,
+		ProcMask:    core.ProcMaskMeleeMHSpecial | core.ProcMaskMeleeMHAuto,
+		Flags:       SpellFlagOmen | core.SpellFlagMeleeMetrics | core.SpellFlagNoOnCastComplete,
+
+		RageCost: core.RageCostOptions{
+			Cost:   rageCost,
+			Refund: 0.8,
+		},
+
+		DamageMultiplier: 1 + .1*float64(druid.Talents.SavageFury),
```

Comment:
Same here with savage fury

Reply:
I'm not sure level scaling applies here. Savage fury scaling should be fine I think

Reply:
Sorry it's been a while so missing some context 😅 but since Savage Fury is additive it should be added to `DamageMultiplierAdditive` instead